support/object/object.go

```diff
@@ -0,0 +1,261 @@
+package object
```

Comment:
S: Sorry for the change, I checked other packages, and the `maps` name may be better.

Reply:
No problem, I will change it to maps.